src/Extensions/SiteConfigExtension.php

```diff
@@ -24,8 +24,8 @@ public function updateCMSFields(FieldList $fields)
 
                 LiteralField::create(
                     'GetSiteControlAPIHelp',
-                    'Go to: <a target='_blank' href="https://dash.getsitecontrol.com/">GetSiteControl Dashboard</a>'
-                ),
+                    "Go to: <a target='_blank' href='https://dash.getsitecontrol.com/'>GetSiteControl Dashboard</a>"
+                )
```

Comment:
Always use single quotes in PHP.

Reply:
ok. I need to add double quotes inside the href and target tags though, otherwise won't work.